### ISSUE TITLE
LocalForage duplicated in iOS

### ISSUE THREAD

**Issue by orzsikodon** [localForage/localForage#835]
It seems that when I use localForage v1.7.1 on my ionic 1/cordova iOS app, the localForage dataStore is duplicated. The data they contain is identical, but I don't understand why the debugger shows two instances. 

<img width="914" alt="screen shot 2018-07-17 at 12 36 59 pm" src="https://user-images.githubusercontent.com/6364464/42812496-249e8f4e-89be-11e8-9c96-97e59544fe0c.png">

Any thoughts?

**Comment by tofumatt** [comment]
That’s very strange. It shouldn’t be possible, from what I understand, to create two stores with the same name.

If you manually edit a value in one (either with the developer tools or manually with an IndexedDB call) does it reflect in the other store? I’m wondering if this is a bug with Safari and not localForage.

**Comment by orzsikodon** [comment]
Yes it does! I'm not sure where the issue lies either? How could we show that the issue is in fact with Safari?

**Comment by thgreasi** [comment]
I guess if it only happens on Safari and there are at least two other browsers that this doesn't happen, then it's probably a Safari issue. AFAIK LF doesn't have code that could do the key-value-pair reflection that you observe.

**Comment by tofumatt** [comment]
Again: what happens when you manually change/edit an item in the store? Is it reflected across both stores? Maybe it's a devtools bug.
